### PR TITLE
adding Rangelov's Ahamb Phd thesis

### PR DESCRIPTION
**Pull request by HedvigS**
There now exists a grammar for Ahamb [axam1237], hurray.

### CODE REVIEW

**Comment by xrotwang** [comment]
I'll ask him whether he can upload the Thesis on Zenodo as well. academia URLs only work for members.

**Comment by HedvigS** [comment]
[It's not on Waikato's website from what I can see, yet](https://researchcommons.waikato.ac.nz/browse?type=author&value=Rangelov%2C+Tihomir) but I imagine it'll turn up there sooner or later as well.

**Comment by HedvigS** [comment]
@tihomirrangelov would also like to suggest that the main name for the language is given as "Ahamb" rather than "Axamb". Speakers dislike "Axamb" and it is not the name used by the now most extensive source, i.e. his PhD thesis.

**Comment by xrotwang** [comment]
There's a Zenodo DOI now, so
```bibtex
phdthesis{g:Rangelov:Ahamb,
  author     = {Rangelov, Tihomir},
  pages      = {iv+408},
  school     = {University of Waikato},
  title      = {A grammar of the Ahamb language (Vanuatu)},
  year       = {2020},
  hhtype     = {grammar},
  inlg       = {English [eng]},
  lgcode     = {Ahamb = [axam1237]},
  url        = {https://doi.org/10.5281/zenodo.4601356},
  doi        = {10.5281/zenodo.4601356},
  macro_area = {Papunesia}
}
```

File: references/bibtex/hh.bib [hunk 1]
```diff
@@ -882931,3 +882931,23 @@ @book{s:Laribyan:Hayeren:1958
...
+@phdthesis{ahamb_rangelov,
+  author     = {Rangelov, Tihomir},
+  pages      = {iv+408},
+  school     = {University of Waikato},
+  title      = {A grammar of the Ahamb language (Vanuatu)},
+  year       = {2020},
+  hhtype     = {grammar},
+  inlg       = {English [eng]},
+  lgcode     = {Ahamb = [axam1237]},
+  url ={https://www.academia.edu/44669830/A_grammar_of_the_Ahamb_language_Vanuatu_},
```
Comment (xrotwang) [inline references/bibtex/hh.bib:882945]: @HedvigS you should also change the URL to the one based on the DOI, I think, i.e. `https://doi.org/...`.

**Review by xrotwang — CHANGES_REQUESTED** [review]

File: references/bibtex/hh.bib (hunk 1 above)
Comment (HedvigS) [reply references/bibtex/hh.bib:882945]: Okay, sure. Whatever the standard is for Harald's bibs

**Pull request merged**